README.md

```diff
@@ -17,6 +17,7 @@ The plugin can consume the following report types:
 -   Functional test reports generated by Parasoft SOAtest 9.x and 2020.1+.
```

Comment:
The version requirements is inconsistent with documentation(https://docs.parasoft.com/display/FINDS/Parasoft+Findings+for+Jenkins#ParasoftFindingsforJenkins-Introduction). Should we make them consistent?